dashboards/MySQL_Replication_Summary.json

```diff
@@ -667,6 +691,107 @@
             ],
             "valueName": "current"
         },
+        {
+            "cacheTimeout": null,
+            "colorBackground": false,
+            "colorValue": false,
+            "colors": [
+                "rgba(50, 172, 45, 0.97)",
+                "rgba(237, 129, 40, 0.89)",
+                "rgba(245, 54, 54, 0.9)"
+            ],
+            "datasource": "Prometheus",
+            "decimals": 0,
+            "description": "MySQL supports delayed replication such that a replica server deliberately executes transactions later than the source by at least a specified amount of time.\n\nThe default replication delay is 0 seconds. Use the CHANGE MASTER TO MASTER_DELAY=N statement to set the delay to N seconds. A transaction received from the source is not executed until at least N seconds later than its commit on the immediate source. \n\nSee also:",
```

Comment:
Should there be something after `See also:`, or is it by design so?

Reply:
"See also" is followed by a link
![Screenshot_20201029_110534](https://user-images.githubusercontent.com/25184101/97547728-d5776700-19d6-11eb-8176-d32daac45fce.png)

Reply:
Sure, makes sense!